fastlane/lanes/localization.rb

```diff
@@ -327,6 +330,7 @@ def download_localized_app_store_metadata(glotpress_project_url:, locales:, meta
           + "Delete the `#{en_file_path}` file, ensure the `default/` one has the expected original copy, and try again.")
       end
     end
+    # rubocop:enable Metrics/AbcSize
```

Comment:
This seems to be at the wrong place. The corresponding `rubocop:disable` on line 302 is at the same level as the function declaration (indentation level 1), while this balancing one is inside the function body (indentation level 2).

We should either move the `disable` on line 305 inside the function body, or move the `enable` on line 353 after the `end` of the function body.

Reply:
Good catch, thanks! Clearly went for the wrong `end` 🤦‍♂️ Will move to the end of the method.